### PR TITLE
tests/ci: add grafiti trigger as jenkins code

### PR DESCRIPTION
**Pull request by cpanato**
as part of INST-84

cc @mxinden

### CODE REVIEW

File: jenkins-jobs/grafiti_pr_trigger.groovy [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      commentFilePath("")
+      skipBuildPhrase(".*\\[skip\\W+ci\\].*")
+      blackListCommitAuthor("")
+      allowMembersOfWhitelistedOrgsAsAdmin(true)
+      msgSuccess("")
+      msgFailure("")
+      commitStatusContext("Jenkins-Tectonic-Installer")
+      buildDescTemplate("#\$pullId: \$abbrTitle")
+      blackListLabels("")
+      whiteListLabels("run-smoke-tests")
```
Comment (mxinden) [inline jenkins-jobs/grafiti_pr_trigger.groovy]: @cpanato s/"run-smoke-tests"/""/

File: jenkins-jobs/grafiti_pr_trigger.groovy [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      displayBuildErrorsOnDownstreamBuilds(false)
+      commentFilePath("")
+      skipBuildPhrase(".*\\[skip\\W+ci\\].*")
+      blackListCommitAuthor("")
+      allowMembersOfWhitelistedOrgsAsAdmin(true)
+      msgSuccess("")
+      msgFailure("")
+      commitStatusContext("Jenkins-Tectonic-Installer")
+      buildDescTemplate("#\$pullId: \$abbrTitle")
+      blackListLabels("")
```
Comment (mxinden) [inline jenkins-jobs/grafiti_pr_trigger.groovy]: @cpanato Can you add the "do-not-test" label here?

Comment (cpanato) [inline jenkins-jobs/grafiti_pr_trigger.groovy]: done

File: jenkins-jobs/grafiti_pr_trigger.groovy (hunk 1 above)
Comment (cpanato) [inline jenkins-jobs/grafiti_pr_trigger.groovy]: done

**Comment by cpanato** [comment]
adding @estroz

**Comment by estroz** [comment]
@cpanato squash your commits into one and LGTM

**Comment by cpanato** [comment]
@estroz done

**Pull request merged**